docs/hardhat-network/explanation/mining-modes.md

```diff
@@ -25,6 +27,24 @@ const pendingBlock = await network.provider.send("eth_getBlockByNumber", [
 ]);
 ```
 
+### Mining transactions in FIFO order
+
+The way Hardhat Network's mempool orders transactions is customizable. By default, they are prioritized following Geth's rules, but you can enable a FIFO mode instead. This mode ensures that transactions are added to blocks in the same order they are sent, and is useful to recreate blocks from other networks.
```

Comment:
Maybe I'm picking too small of a nit here, but I'd prefer to leave the word "mode" out of this, since we're already formalizing that word with the "Mining Modes" article title.  I'd prefer to keep using "behavior" here, for the sakes of consistency and differentiation.

```suggestion
The way Hardhat Network's mempool orders transactions is customizable. By default, they are prioritized following Geth's rules, but you can enable a FIFO behavior instead, which ensures that transactions are added to blocks in the same order they are sent, and which is useful to recreate blocks from other networks.
```